test/EVMHost.cpp

```diff
@@ -418,15 +440,19 @@ evmc::result EVMHost::precompileECRecover(evmc_message const& _message) noexcept
 				"debaaa0cddb321b2dcaaf846d39605de7b97e77ba6106587855b9106cb104215"
 				"61a22d94fa8b8a687ff9c911c844d1c016d1a685a9166858f9c7c1bc85128aca"
 			),
-			fromHex("0000000000000000000000008743523d96a1b2cbe0c6909653a56da18ed484af")
+			{
+				fromHex("0000000000000000000000008743523d96a1b2cbe0c6909653a56da18ed484af"),
+				gas_cost
+			}
 		}
 	};
 	evmc::result result = precompileGeneric(_message, inputOutput);
 	// ECRecover will return success with empty response in case of failure
-	if (result.status_code != EVMC_SUCCESS)
+	if (result.status_code != EVMC_SUCCESS && result.status_code != EVMC_OUT_OF_GAS)
+//		return resultWithGas(_message.gas, gas_cost, {});
```

Comment:
Doesn't look like this belongs here given that we want result to be success in this case.

Please remove or add a comment explaining why it's here.

Reply:
I actually wanted to use this but then had concerns the new `{}` object used as `const&` could cause issues. If it is fine we can use that shorthand, otherwise just drop the line.

Reply:
Ah, right. Just realized that the function sets success as well so it's equivalent after all (I've been staring at this code too long :)).

Yeah, the `const&` might be a concern - even if not an actual problem, we had cases of sanitizers complaining about that.

In any case, it's fine for me either way, just with a short comment that it's equivalent if you leave this commented out.

Reply:
I'd just remove it, but also don't want to wait for CI. If we are confident that #13532 is merged soon we could remove the commented out line there? 😅